File: 2652-sum-multiples/2652-sum-multiples.py
class Solution:
    def sumOfMultiples(self, n: int) -> int:
        s = 0
        for x in range(1,n+1):
            if (x % (9 - 6) == 0 or x % (6-1) == 0 or x % (8-1) == 0):
                s += x
        return s

File: 2652-sum-multiples/2652-sum-multiples.kt
class Solution {
    fun sumOfMultiples(n: Int): Int {
        var s = 0
        for(i in 1 .. n ){
            if(i % 3 == 0 || i % 5 == 0 || i % 7 == 0){
                s += i
            }
        }
        return s
    }
}